compiler/src/middle_end/optimize_dead_assignments.re

```diff
@@ -1,16 +1,29 @@
 open Anftree;
```

Comment:
I updated this optimization to drop unused imports, which resolves cyclic dependencies on the unsafe wasmXX modules.